web/src/components/atomic/SyntaxHighlight.ts

```diff
@@ -0,0 +1,34 @@
+import '~/style/prism.css';
+
+import Prism from 'prismjs';
+import * as Vue from 'vue';
+import { VNode } from 'vue';
```

Comment:
I would prefer using named imports instead of a wildcard import.

```suggestion
import { h, defineComponent, VNode } from 'vue';
```

Reply:
done